run_dir/design/bioinfo_tab/run_lane_sample_view.html

```diff
@@ -1,6 +1,6 @@
   <table class="table table-bioinfo-status {% if project_closed %} bioinfo-status-disabled {% end %} table-bioinfo-status-runview table-bordered">
```

Comment:
There's something funky going on with the default table, it's not respecting the container width, while if you switch to sample-run-lane, it does. I don't think this is due to this PR, since there is something slightly weird but in a different direction going on on what's already in production. But maybe you have a chance to correct it now? Not a dealbreaker though, it's just slightly annoying.

Reply:
I think its because the column `Sample` in the default table contains long flowcell names chained with underscores which the table does not break. Its the same in the other table when it is expanded to show the FC names.